File: src/main/java/com/teamwizardry/librarianlib/book/util/Page.java
package com.teamwizardry.librarianlib.book.util;

import com.teamwizardry.librarianlib.book.gui.GuiBook;

public class Page {

	public final String path;
	public final int page;
	public GuiBook gui;
	
	public Page(String path, int page) {
		this.path = path;
		this.page = page;
	}
	
	public Page pageNum(int page) {
		return new Page(path, page);
	}
}

File: src/main/java/com/teamwizardry/librarianlib/book/util/Page.kt
package com.teamwizardry.librarianlib.book.util

import com.teamwizardry.librarianlib.book.gui.GuiBook

class Page(val path: String, val page: Int) {
    var gui: GuiBook? = null

    fun pageNum(page: Int): Page {
        return Page(path, page)
    }
}
